routingapi/routingapi_test.go

```diff
@@ -18,6 +18,10 @@ import (
 	fakeuaa "code.cloudfoundry.org/route-registrar/routingapi/routingapifakes"
 )
 
+func ptrOfInt(i uint16) *uint16 {
```

Comment:
This change is not necessary to merge, but I would consider renaming this to `pointerToInt` or `ptrToInt` to make this more clear.